lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -463,6 +463,16 @@ class ReportUIFeatures {
     this._document.removeEventListener('keydown', this.onKeyDown);
   }
 
+  /**
+   * @param {Function} cb
+   */
+  _setLightThemeTemporarily(cb) {
+    const isDark = this._dom.find('.lh-vars', this._document).classList.contains('dark');
```

Comment:
~~yikes 😆 I wonder if we should use a variable or something in ReportUIFeatures state...~~

misread and realized this was testing `classList` and not `textContent` but a function or something that encapsulates this might be nice